pennylane/_grad.py

```diff
@@ -288,9 +367,60 @@ def circuit(x, y):
 
     As expected, the tuple was unpacked and we directly received the Jacobian of the
     QNode with respect to ``y``.
+
```

Comment:
We can leave it out of this PR, but it's a shame `grad` doesn't have any examples (not just for `qjit`).